uaclient/contract.py

```diff
@@ -25,7 +29,7 @@ def __init__(self, e, error_response):
         else:
             self.api_errors = [error_response]
         for error in self.api_errors:
-            error['code'] = error['title']
+            error['code'] = error.get('title', error.get('code'))
```

Comment:
why would the error have and why are we replacing it with title if present?
if 'title in error:
error['code'] = error['title']
?